### DISCUSSION TITLE
Question about 'index_cpu_to_gpus_list'

### DISCUSSION THREAD

**Discussion by ezeli** [facebookresearch/faiss#3564] [General]
# Summary

I have two GPUs: 0 and 1. When I use `faiss.index_cpu_to_gpus_list(index, gpus=[1])` to specify GPU 1, GPU 0 is also used. What is the data on GPU 0? How can I only use GPU 1?
![image](https://github.com/facebookresearch/faiss/assets/28505751/e9c4ce0b-702e-43ec-b396-12b8e4cf0527)
![image](https://github.com/facebookresearch/faiss/assets/28505751/75c6a7eb-6306-4a05-afe1-6d4da0f2850c)

# Platform

OS: CentOS Linux

Faiss version: 1.7.2

Installed from: pip

Faiss compilation options: 

Running on:
- [ ] CPU
- [x] GPU

Interface: 
- [ ] C++
- [x] Python

# Reproduction instructions
```python
import faiss
import numpy as np
vectors=np.random.rand(2000000, 312).astype(np.float32)
d = vectors.shape[1]
index = faiss.IndexFlatIP(d)
index = faiss.index_cpu_to_gpus_list(index, gpus=[1])
```

**Comment by shawnye2000** [comment]
Same issue, have you solve this issue?
